docs/content/docs/dev/table/functions/udfs.md

```diff
@@ -647,15 +647,15 @@ The following system functions are always non-deterministic(evaluated per record
 - RAND_INTEGER
 - CURRENT_DATABASE
 - UNIX_TIMESTAMP
+- CURRENT_TIMESTAMP
 - CURRENT_ROW_TIMESTAMP
+- LOCALTIMESTAMP
+- NOW
```

Comment:
These methods are dynamic function, see CurrentTimePointCallGen

Reply:
yes, for the runtime implemantation, they all behavior as dynamic functions, I'll update this